src/assets/server/rest/v1/docs/open-api-standard.json

```diff
@@ -518,4 +705,4 @@
       }
     }
   }
-}
\ No newline at end of file
+}
```

Comment:
The whole old auth endpoints have not been ported to the new API. 
First list all the endpoints in the old API and translate them to the new API scheme.